[GT-133] Adds help information about series exclusion in guide page

rowan04 opened the pull request:
- in guide page, adds help information about series exclusion
- moves two other panels to other side of the page
- adds two images used in the help guide

resolves #150

rowan04 commented:
![seriesexclusionchange](https://user-images.githubusercontent.com/90393428/224691408-78386e61-1b78-4626-ad8f-399508547d58.png)

gregcorbett commented:
Why do the screenshots seem to show missing images for removedseries and addedseries?

rowan04 commented:
> Why do the screenshots seem to show missing images for removedseries and addedseries?

I added those images in in this pull request - as i added them via vscode, they weren't updated and added to the grafana dashboard. At least, that's what i think/hope.
